utils/lib.sh

```diff
@@ -1686,7 +1686,7 @@ LXC_BASE_PACKAGES_fedora="bash git @development-tools python"
 LXC_BASE_PACKAGES_centos="bash git python3"
 
 case $DIST_ID in
-    ubuntu|debian) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
+    ubuntu|debian|raspbian-*) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
```

Comment:
```suggestion
    ubuntu|debian|raspbian) LXC_BASE_PACKAGES="${LXC_BASE_PACKAGES_debian}" ;;
```

Reply:
We're only inspecting $DIST_ID here, not its version.

Reply:
Yes i missed that